client/src/App.js

```diff
@@ -19,9 +20,13 @@ import { OnScreenKeyboardLayout, OnScreenKeyboardDisplay } from './constants/Key
 
 import './App.css';
 import 'react-simple-keyboard/build/css/index.css';
+import CanvasToolbox from './components/toolbox';
+
+const DEFAULT_ROOM = 'main';
```

Comment:
this could pose as a problem later. If we expose room names in plain text, people could manipulate it by modifying the source code. But for now lets leave it like this